cli/ingress-controller/main.go

```diff
@@ -392,14 +404,44 @@ func main() {
 				Client: kongClient,
 			},
 		}
+		var cert tls.Certificate
+		if cliConfig.AdmissionWebhookCertPath != "" && cliConfig.AdmissionWebhookCert != "" {
+			if cliConfig.AdmissionWebhookCertPath != "/admission-webhook/tls.crt" {
+				glog.Fatalf("both --admission-webhook-cert-file and --admission-webhook-cert" +
+					"are set. Please remove one or the other")
+			}
+		}
+		if cliConfig.AdmissionWebhookKeyPath != "" && cliConfig.AdmissionWebhookKey != "" {
+			if cliConfig.AdmissionWebhookKeyPath != "/admission-webhook/tls.key" {
```

Comment:
magic string of `/admission-webhook/tls.key` used in two places with this change.
Please define it as a constant in `flags` package and then use it here.